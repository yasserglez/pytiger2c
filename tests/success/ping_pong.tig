let
    function ping(k: int) = 
        if k > 0 then (print("Ping!\n");  pong(k))
        
    function pong(k: int) =
        (print("Pong!\n"); ping(k - 1))
in
    ping(10)
end
