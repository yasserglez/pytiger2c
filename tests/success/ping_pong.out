Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
Ping!
Pong!
